app/src/test/java/tttTDD/Java/BoardTest.java

```diff
@@ -8,23 +8,25 @@ public class BoardTest {
     Board board;
 
     @Test
-    public void getBoardInstanceWithCorrectLength() {
-        board = new Board();
+    public void getBoardInstanceWithCorrectDefaultLengthOf3() {
+        board = new Board(3);
 
-        Assert.assertEquals(board.getBoard().length, 9);
+        Assert.assertEquals(board.getSpace(0), "1");
+        Assert.assertEquals(board.getSpace(8), "9");
     }
 
     @Test
     public void getBoardInstanceWithResizableBoard() {
         board = new Board(4);
-        Assert.assertEquals(board.getBoard().length, 16);
+
+        // Assert
         Assert.assertEquals(board.getSpace(0), "1");
         Assert.assertEquals(board.getSpace(15), "16");
     }
 
     @Test
     public void getBoardSpace() {
-        board = new Board();
+        board = new Board(3);
 
         Assert.assertEquals(board.getSpace(5), "6");
         Assert.assertNotEquals(board.getSpace(5), "5");
```

Comment:
This second assertion is a bit redundant, unless there's a chance that your getter method also changes the the value of the field.